lib/Chat/SystemMessage/Listener.php

```diff
@@ -408,11 +417,13 @@ protected function sendSystemMessage(Room $room, string $message, array $paramet
 			$referenceId = (string) $referenceId;
 		}
 
-		$this->chatManager->addSystemMessage(
+		return $this->chatManager->addSystemMessage(
```

Comment:
The method `addSystemMessage` haven't docblock explaining what `shouldSkipLastMessageUpdate` do.
The argument name explain what he does but, what is the impact of skip last message update?
Maybe will be good use docblock in `addSystemMessage` or also in `sendSystemMessage`

The `event->shouldSkipLastMessageUpdate` also haven't explanation about what he does.

Reply:
> what is the impact of skip last message update?

It skips the update of the last message field in the database

Reply:
What is the impact when we skip this update? When isn't necessary?

Reply:
Added